pump/storage/log.go

```diff
@@ -338,7 +338,13 @@ func seekToNextRecord(reader *bufio.Reader) (bytes int, err error) {
 			return
 		}
 
-		reader.Discard(1)
+		var discarded int
+		discarded, err = reader.Discard(1)
+		if err != nil {
```

Comment:
At L329
we `reader.Peek(4)` and success, so we should always success to Discard one here.
what about just fatal if `err != nil` here and add some comment

Reply:
I'm not sure. What's the difference it should never happens?